src/ConnectedMode.UnitTests/Suppressions/TimedUpdateHandlerTests.cs

```diff
@@ -103,6 +108,27 @@ public void Dispose_RefreshTimerDisposed_RaisingEventDoesNothing()
             qualityProfileUpdater.Verify(x => x.UpdateAsync(), Times.Never);
         }
 
+        [TestMethod]
+        public void InvokeBindingChanged_TimerStartsStopsOnActiveSolutionBoundChange()
+        {
+            var refreshTimer = new Mock<ITimer>();
+            var timerFactory = CreateTimerFactory(refreshTimer.Object);
+            var activeSolutionBoundTracker = CreateActiveSolutionBoundTrackerWihtBindingConfig(SonarLintMode.Standalone);
+
+            _ = CreateTestSubject(activeSolutionBoundTracker:activeSolutionBoundTracker.Object, timerFactory:timerFactory);
+
+            refreshTimer.Verify(x => x.Start(), Times.Never);
+            refreshTimer.Verify(x => x.Stop(), Times.Once);
```

Comment:
Lines 120 and 121 are effectively checking that the constructor starts/stops the timer based on the initial binding state. However, it's only testing what happens if the solution is initially in standalone mode.

This perhaps should be a separate test i.e. that the constructor starts/stops the time based on the initial binding state. It could be a data-driven test.